wordpress-mobile/WordPress-iOS
# [Site creation] Apply styling to cells in Verticals screen

Implements #10482 

![simulator screen shot - iphone xr - 2018-12-12 at 13 32 29](https://user-images.githubusercontent.com/2722505/49849123-1b42da80-fe13-11e8-9960-c05023b057b2.png)

- [ ] Apply styles to cell
- [ ] Add a subtitle when `SiteVertical.isNew` is true
- [ ] Set the title style to italic when `SiteVertical.isNew` is true

To test:
- Check out the branch
- Build the branch, run the tests.
- Navigate to My Sites > + > Enhanced site creation > Select any kind of site > Start typing in the search field.
- Notice the bottom cell contains a subtitle, and the tile is rendered in italics

Notes:
The search field does not change locations when users start typing. That behaviour is implemented in PR #10494 

Update release notes:

- There are no user facing changes yet, as te feature is behind a feature flag
